From dee15198c9ee17f18ac631cea2a3991e7dcf7ed0 Mon Sep 17 00:00:00 2001
From: Adrian Freund <adrian@freund.io>
Date: Sat, 12 Oct 2024 11:16:16 +0200
Subject: [PATCH] Change imports from preview to typst-forge

---
 lib.typ              | 2 +-
 template/example.typ | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib.typ b/lib.typ
index 9e03466..835e1f7 100644
--- a/lib.typ
+++ b/lib.typ
@@ -1,4 +1,4 @@
-#import "@preview/fontawesome:0.2.1": *
+#import "@typst-forge/fontawesome:0.2.1": *

 #let _cv-line(left, right, ..args) = {
   set block(below: 0pt)
diff --git a/template/example.typ b/template/example.typ
index f84d693..df39fb2 100644
--- a/template/example.typ
+++ b/template/example.typ
@@ -1,4 +1,4 @@
-#import "@preview/moderner-cv:0.1.0": *
+#import "@typst-forge/moderner-cv:0.1.0": *

 #show: moderner-cv.with(
   name: "Jane Doe",
--
2.46.2
