From 4ea9991599ba7a3cee003aafb616306260d94c06 Mon Sep 17 00:00:00 2001
From: Adrian Freund <adrian@freund.io>
Date: Sat, 12 Oct 2024 11:16:16 +0200
Subject: [PATCH] Change template file import to typst-forge

---
 template/example.typ | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/template/example.typ b/template/example.typ
index f84d693..df39fb2 100644
--- a/template/example.typ
+++ b/template/example.typ
@@ -1,4 +1,4 @@
-#import "@preview/moderner-cv:0.1.0": *
+#import "@typst-forge/moderner-cv:0.1.0": *

 #show: moderner-cv.with(
   name: "Jane Doe",
--
2.46.2
